charts/seed-controlplane/charts/csi-alicloud/templates/csi-plugin-dep.yaml

```diff
@@ -22,6 +22,10 @@ spec:
         garden.sapcloud.io/role: controlplane
         app: kubernetes
         role: csi-plugin-controller
+        networking.gardener.cloud/to-dns: allowed
+        networking.gardener.cloud/to-public-networks: allowed
+        networking.gardener.cloud/to-private-networks: allowed
```

Comment:
I didn't see it need to access private networks. Would you please confirm that?

Reply:
Not needed